docs/src/filters/writing_custom_filters.md

```diff
@@ -282,7 +316,7 @@ let config = match args.config.unwrap() {
 
 The [Dynamic][ConfigType::dynamic] contains the serialized [Protobuf] message received from the [management server] for the [Filter] to create.
 As a result, its contents are entirely opaque to Quilkin and it is represented with the [Prost Any][prost-any] type so the [FilterFactory]
-can interpret its contents however it wishes.  
+can interpret its contents however it wishes.
```

Comment:
Haven't got a better spot to put this, but all the documentation about Dynamic Config vs Static should be changed to align with Text/Binary configuration.